src/fetch_civil_names.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+import os
+import requests
+import re
+import pandas as pd
+import numpy as np
+from bs4 import BeautifulSoup
+
+
+DATA_BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Actually this is the base dir of the project, isn't it? Should we call it `BASE_DIR`?